### PR TITLE
[CodeClean] merge get-model scripts

### PR DESCRIPTION
**Pull request by mojunsang26**
Delete the existing get-model scripts and merge it into get-model.sh
Identify models by arguments

related issue: #64 

Signed-off-by: Junsang Mo <junsang.mo@samsung.com>

### CODE REVIEW

File: bash_script/example_models/get-model-image-classification-tflite.sh [hunk 1]
```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
-mkdir -p tflite_model_img
-cd tflite_model_img
-wget https://storage.googleapis.com/download.tensorflow.org/models/tflite/mobilenet_v1_1.0_224_quant_and_labels.zip
```
Comment (jaeyun-jung) [inline bash_script/example_models/get-model-image-classification-tflite.sh]: In TAOS CI, NNStreamer use this script to download the test model in app-test module. You should consider to change this also.

https://github.com/nnsuite/TAOS-CI/blob/f8503d4fb3c850b0f1c87be9a71189f04c7f4d62/ci/taos/plugins-staging/pr-postbuild-nnstreamer-ubuntu-apptest.sh#L185

Comment (myungjoo) [reply bash_script/example_models/get-model-image-classification-tflite.sh]: Or you may add a symlink and use the first param to determine which symlink was used to call it. Then you can skip updating TAOS CI and make it backward compatible.

File: bash_script/example_models/meson.build [hunk 2]
```diff
@@ -1,4 +1,3 @@
-install_data(['get-model-object-detection-tf.sh', 'get-model-object-detection-tflite.sh',
```
Comment (jaeyun-jung) [inline bash_script/example_models/meson.build]: It needs to install get-model-image-classification-tflite.sh for TAOS CI.

Comment (mojunsang26) [reply bash_script/example_models/meson.build]: Thank you for your guidance! I updated a meson.build script. Now, TAOS CI can use get-model-image-classification-tflite.sh as usual.

**Pull request merged**